coreLib/src/main/java/dev/kosmx/playerAnim/core/data/gson/AnimationJson.java

```diff
@@ -122,17 +122,28 @@ private void moveDeserializer(KeyframeAnimation.AnimationBuilder emote, JsonArra
             JsonObject obj = n.getAsJsonObject();
             int tick = obj.get("tick").getAsInt();
             String easing = obj.has("easing") ? obj.get("easing").getAsString() : "linear";
+            Float easingArg = null;
+            try {
+                if (obj.has("easingArg")) {
+                    easingArg = easingArg.floatValue();
```

Comment:
I think you're wrong here... As I see it `easingArg` is always null.

Reply:
Well yeah usually it is
But I am assuming if you encode a Blockbench animation to the Emotecraft format then it will have it?

Reply:
I mean something like this:

```suggestion
                    easingArg = obj.get("easingArg").getAsFloat();
```